rpc_jobs/rpc_openstack.yml

```diff
@@ -167,10 +262,10 @@
       - 'PM-push-trigger_{repo_name}-{branch}-{image}-{scenario}-{action}'
 
 - project:
-    name: "rpc-openstack-master-mnaio-postmerge"
+    name: "rpc-openstack-pike-mnaio-postmerge"
```

Comment:
I think it might make sense to add the pike job and keep the master one, however i've no strong opinions on this; if we need it we could always add it back.

Reply:
The master job starts at line 214.